code/modules/mob/living/carbon/xenomorph/castes/Facehugger.dm

```diff
@@ -140,12 +140,10 @@
 	var/did_hug = hugger.attach(human, TRUE, 0.5)
 	if(client)
 		client?.player_data?.adjust_stat(PLAYER_STAT_FACEHUGS, STAT_CATEGORY_XENO, 1)
+	xeno_message(SPAN_XENOMINORWARNING("You sense that [src] has facehugged a host at [loc.loc.name]!"), 1, src.hivenumber)
```

Comment:
loc.loc is risky, better to just do get_area and then check if get_area actually exists, and then putting them messages here below that if check

Reply:
got distracted by tubes